services/drupal/web/themes/epa_theme/source/_patterns/05-components/document/document.twig

```diff
@@ -0,0 +1,9 @@
+<span class="document">
+  {{ file }}
+  <span class="document__meta">
+    {% if date or publication_number %}
```

Comment:
We also want to see what a document with description looks like in Pattern Lab, so let’s also create a `document~with-description.yml` file with the description variable also set.

Reply:
@dcmouyard I didn't create a specific variation for the with description because both view modes would use the same component. If any of the date, ID, or description doesn't exist they just don't display. If it's something we do need to show separately I can drop that in.

Reply:
Yeah, I just wanted to showcase both examples in Pattern Lab.